blog/2025-06-05.md

```diff
@@ -0,0 +1,75 @@
+---
+title: '📢 Milestone 0.5 has been reached'
+authors: mayya
+tags: [tanager, release]
+---
+
+We’re excited to announce the **0.5 release** milestone for the [Ed-Fi Data
+Management Service
+(DMS)](https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/), marking
+another step forward on our path toward the upcoming **Release Candidate 1**.
+This release delivers key features to support authorization, extensibility, and
+interoperability.
+
+<!-- truncate -->
+
+## Key Updates
+
+1. **MetaEd-Based Extensions**: Adds support for TPDM and custom extensions
+   authored using MetaEd.
+2. **SDK Generation**: Automates client SDK generation to facilitate client
+   application development.
+3. **Dependency Endpoint & XSD API**: Supports XML bulk load testing and
+   compatibility validation.
+4. **Claimset-Based Authorization Metadata**: DMS Configuration Service* now
+   includes claimset-based authorization metadata endpoint to facilitate request
+   authorization in Data Management Service.
+5. **Relationship-Based Authorization**: Enables access control via [student
+   enrollment](https://github.com/Ed-Fi-Alliance-OSS/Project-Tanager/blob/main/docs/DMS/AUTHORIZATION/DOCUMENT-AUTHORIZATION/STUDENT-EDORG-AUTHORIZATION.md)
+   and [staff
+   employment](https://github.com/Ed-Fi-Alliance-OSS/Project-Tanager/blob/main/docs/DMS/AUTHORIZATION/DOCUMENT-AUTHORIZATION/STAFF-EDORG-AUTHORIZATION.md).
+
+*Note: DMS Configuration Service is a separate component designed to manage
+claimsets, authorization metadata, and configuration data for DMS deployments.
+
+📌[View Full
+Changelog](https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/compare/v0.4.0...v0.5.0)
+
+## Try It Out
+
+With Docker Desktop or Podman installed locally:
+
+```powershell
+git clone https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service
+cd Data-Management-Service
+git checkout v0.5.0
+
+# Start all required services inside of Docker
+cd eng\docker-compose
+cp .env.example .env
+.\start-local-dms.ps1 -EnableSearchEngineUI -EnableConfig -r
+```
+
+The `.http` files in `src/dms/tests/RestClient` can help you understand how to
+interact with these systems. In particular, see
+`authorization-demonstration-Dec2024.http`. These files should be executable in
```

Comment:
Might want to direct people to https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/blob/main/GETTING_STARTED.md instead, which uses a different (newer) `.http` file.